nsqadmin/static/js/main.js

```diff
@@ -14,8 +14,18 @@ require('./lib/ajax_setup');
 require('./lib/handlebars_helpers');
 
 var start = function() {
-    new AppView();
-    Router.start();
+    $.ajax({
```

Comment:
there should be a way for us to inject this data into the template rather than create a new endpoint and have to perform this AJAX request